### PR TITLE
[docs] add links to the azure devops project in the general developer documentation

### PR DESCRIPTION
**Pull request by paule96**
Hi,

here are the documentation for the links to the azure devops stuff.

### CODE REVIEW

File: docs/pages/development-guides/general-guide.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ This is the current project structure.
 
 > If you need a tutorial about important git commands look [here](https://dev.to/dhruv/essential-git-commands-every-developer-should-know-2fl).
 
+> If you want see the CI status of your PR you can take a look at [this Azure DevOps page](https://dev.azure.com/T-Systems-MMS/phonebook/_build). There are all builds that are running. To take a look at the current release you can look at [this page](https://dev.azure.com/T-Systems-MMS/phonebook/_release). (that are only docker hub releases)
```
Comment (mschwrdtnr) [inline docs/pages/development-guides/general-guide.md]:
```suggestion
> If you want to see the CI status of your PR you can take a look at [our Azure DevOps Page](https://dev.azure.com/T-Systems-MMS/phonebook/_build). There are all running builds. To take a look at the current release you can [view this page](https://dev.azure.com/T-Systems-MMS/phonebook/_release) (this are only docker hub releases).
```
What do you think about that?

**Comment by mschwrdtnr** [comment]
Resolves #193

**Comment by DanielHabenicht** [comment]
Can we also add a badge on the readme.md?

File: docs/pages/development-guides/general-guide.md (hunk 1 above)
Comment (DanielHabenicht) [inline docs/pages/development-guides/general-guide.md]: Those are not docker hub releases, that's just the preview (demo-phonebook.me) beeing released. 

The Hub Images are pushed in  the production build.

**Comment by paule96** [comment]
maybe I should create a separate build and release page.

**Comment by DanielHabenicht** [comment]
ping @mschwrdtnr

**Pull request merged**